tests/openvino/test_quantization.py

```diff
@@ -490,6 +490,27 @@ def test_ovmodel_load_large_model_with_uncompressed_weights(self):
                     }
                     save_model_patch.aasert_called_with(saving_params)
 
+    def test_ovmodel_load_large_model_with_additional_quantization_config(self):
+        with unittest.mock.patch("transformers.modeling_utils.ModuleUtilsMixin") as model_mixin_patch:
+            model_mixin_patch.num_parameters.return_value = 2e9
+            with unittest.mock.patch("openvino.runtime.ie_api.Core.read_model") as core_patch:
+                with unittest.mock.patch("optimum.exporters.openvino.convert._save_model") as save_model_patch:
+                    _ = OVModelForCausalLM.from_pretrained(
+                        MODEL_NAMES["llama"],
+                        export=True,
+                        compile=False,
+                        use_cache=False,
+                        quantization_config=OVWeightQuantizationConfig(bits=4, sym=True, group_size=-1, ratio=0.8),
+                    )
+                    # quantization will be performed later, using load_model
+                    saving_params = {
+                        "model": unittest.mock.ANY,
+                        "path": unittest.mock.ANY,
+                        "compression_option": "fp32",
+                        "compression_ratio": None,
```

Comment:
should be replaced with `ov_config`

Reply:
I think it should be take care in separated pr with other tests, I keep it in the same way how it was used in other tests

Reply:
I want to secure a case in  which I met an issue (I tried to compress the model to int4 and noted that the model was compressed into int8 instead).